src/bitcoin-cli.cpp

```diff
@@ -1059,7 +1059,13 @@ static void ParseGetInfoResult(UniValue& result)
         result_string += "\n";
     }
 
-    result_string += strprintf("%sWarnings:%s %s", YELLOW, RESET, result["warnings"].getValStr());
+    const std::string warnings{result["warnings"].getValStr()};
+    if (warnings.empty()) {
+        result_string += strprintf("%sWarnings: %s(none)", YELLOW, RESET);
```

Comment:
```suggestion
        result_string += strprintf("%sWarnings:%s (none)", YELLOW, RESET);
```
------
alternatively, could simplify

```diff
-    if (warnings.empty()) {
-        result_string += strprintf("%sWarnings: %s(none)", YELLOW, RESET);
-    } else {
-        result_string += strprintf("%sWarnings:%s %s", YELLOW, RESET, warnings);
-    }
+    result_string += strprintf("%sWarnings:%s %s", YELLOW, RESET, warnings.empty() ? "(none)" : warnings);
```

Reply:
Done in last commit.